perfkitbenchmarker/linux_benchmarks/cloudharmony_network_benchmark.py

```diff
@@ -139,6 +139,12 @@ def _PrepareServer(vm):
     vm: the server virtual machine.
   """
   vm.Install(NGINX)
+
+  # Required to make uplink test function properly
+  vm.RemoteCommand('sudo sed -i "/server_name _;/a error_page  405 =200 $uri;" /etc/nginx/sites-enabled/default')
```

Comment:
why do you need to set the error_page flag? Also does this set error to return 200? (not error)

Reply:
This was the solution I found in several places for enabling POST requests on static pages with nginx. Honestly, it seems kinda hacky, but it works to allow the uplink test to function properly.